src/channels.ts

```diff
@@ -51,17 +51,17 @@ export default function (app: Application): void {
 
     console.log(hook)
 
-    console.log('Publishing all events to all authenticated users. See `channels.js` and https://docs.feathersjs.com/api/channels.html for more information.') // eslint-disable-line
+    console.log('Publishing all events to all authenticated user. See `channels.js` and https://docs.feathersjs.com/api/channels.html for more information.') // eslint-disable-line
 
-    // e.g. to publish all service events to all authenticated users use
+    // e.g. to publish all service events to all authenticated user use
```

Comment:
Heh, so, this is fine to leave as is, but a good example of why an unattended/non-interactive search and replace leads to grammar weirdness. :)